.github/workflows/yaml-checks.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: yamllint
+on: [push, pull_request]
+
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/containers/os-autoinst_dev
+    steps:
+      - uses: actions/checkout@v4
+      - name: Validate yamls
+        run: |
+          git config --global --add safe.directory '*'
+          yamllint --strict $(git ls-files "*.yml" "*.yaml" 2> /dev/null || find . -name '*.y*ml')
```

Comment:
This would be good to have in a makefile, or script or wherever currently style checks for this repo are defined. So one can do 'make check' or sth and include this and other checks

Reply:
But there is not a makefile on this repo!

Reply:
I suggest to create one then, like in os-autoinst-common :)
Also you could do the same as what @okurz adds to the Makefile in https://github.com/os-autoinst/os-autoinst-common/pull/49